controllers/placementapi_controller.go

```diff
@@ -851,8 +1004,14 @@ func (r *PlacementAPIReconciler) reconcileNormal(ctx context.Context, instance *
 	//
 
 	// Define a new Deployment object
-	deplDef := placement.Deployment(instance, inputHash, serviceLabels, serviceAnnotations)
+	deplDef, err := placement.Deployment(ctx, helper, instance, inputHash, serviceLabels, serviceAnnotations)
```

Comment:
``deplDef, err =`` ?

Reply:
meant doing the same as you did [above](https://github.com/openstack-k8s-operators/placement-operator/pull/92/files#diff-c0f1c404ea10bc348fba06aab6f455b95b7513a337e09b325ebea615fc0ee587R984)

Reply:
I see, the `deplDef` has not been declared before, apart from `ctrlResult` as used [above](https://github.com/openstack-k8s-operators/placement-operator/pull/92/files#diff-c0f1c404ea10bc348fba06aab6f455b95b7513a337e09b325ebea615fc0ee587R984).